src/components/Weather.tsx

```diff
@@ -20,11 +20,13 @@ const Weather: React.FC<ComponentProps> = ({ weather, setCityName }) => {
   if (!weather) return null;
 
   return (
-    <Grid container justifyContent="center" spacing={2}>
+    <Grid container justifyContent="center" spacing={2} marginTop={10}>
       <Grid container xs={12} justifyContent="center" direction="row">
         <form
           onSubmit={(event) => {
             event.preventDefault();
+            // should calling submit event here for better UX on Enter keyboard after filling the form. 
+            setCityName(value);
```

Comment:
Try pressing Enter after filling in the City Input. You will see the difference.